src/ui/VideoDrawer.js

```diff
@@ -34,8 +38,9 @@ class VideoDrawer extends React.Component {
       return vidIds;
     } else {
       // if playingSong exists
-      // call api for video id using song id
-      return ['-9fC6oDFl5k', 'WehhSc1knYY'];
+      // call api for video id using song id)
+      return ['0', '0'];
```

Comment:
you can make it an array of just one element: ['0']. I had added 2 video ids to the array to check that they'll play one after another.